POC-OOBE/OOBE-Sandbox/PowerToys Settings Sandbox/Services/ActivationService.cs

```diff
@@ -21,6 +25,88 @@ internal class ActivationService
 
         private object _lastActivationArgs;
 
+        private void newNotification()
```

Comment:
Please add desc for what this notification is / what does it do [or what it will do]

Reply:
Class called NotificationServices was created with a function AppInstalledToast